src/assets/TestFramework/Extension.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.TestFramework
```

Comment:
The namespace here might be incorrect - this class should come from `Azure.ResourceManager.Resources`